src/__internal__/focus-trap/focus-trap-utils.ts

```diff
@@ -5,6 +5,9 @@ type CustomRefObject<T> = {
 const defaultFocusableSelectors =
   'button:not([disabled]), [href], input:not([type="hidden"]):not([disabled]), select:not([disabled]), textarea:not([disabled]), [tabindex]';
 
+const defaultScrollableSelectors =
+  'div[data-role="sidebar-content"], div[data-role="dialog-content"], div[data-role="dialog-full-screen-content"]';
```

Comment:
suggestion (non-blocking): it might be worth using `data-component` here so we're consistent with the other DOM query based implementations we have